app/controllers/signatures_controller.rb

```diff
@@ -19,6 +19,10 @@ def create
     end
   end
 
+  def signed
+    @petition = Petition.find(params[:petition_id])
```

Comment:
I think we'd need to keep it in signature as we use the signature's constituency method? Will go with what Otto suggested